### PR TITLE
Handler and tests

### PR DESCRIPTION
**Pull request by lightning-like**

### CODE REVIEW

File: async_blp/handler_refdata.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+from typing import List
+
+try:
+    import blpapi
+except:
+    from tests import env_test as blpapi
+
+
+class HandlerRef:
+    connection = False
```
Comment (Tymeade) [inline async_blp/handler_refdata.py]: Shouldn't it be instance variable?

File: tests/env_test.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import threading
+import time
+from typing import List
+
+
+class Msg:
```
Comment (Tymeade) [inline tests/env_test.py]: This class is called `Message` in blpapi (just to be consistent with the original blpapi)

File: async_blp/handler_refdata.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    service_name = "//blp/refdata"
+    request_name = "ReferenceDataRequest"
+
+    def __init__(self):
+        self.loop = asyncio.get_running_loop()
+        self.event = asyncio.Event()
```
Comment (Tymeade) [inline async_blp/handler_refdata.py]: I suggest we rename this var to avoid confusion with `blpapi`

File: async_blp/handler_refdata.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        session_options.setServerPort(8194)
+        self.__result = []
+
+        self.session = blpapi.Session(options=session_options,
+                                      eventHandler=self)
+        self.session.startAsync()
+
+    def send_requests(self, requests: List):
+        self.requests = requests
+        self.event.clear()
```
Comment (Tymeade) [inline async_blp/handler_refdata.py]: Probably this should be done using `call_soon_threadsafe`

Comment (lightning-like) [reply async_blp/handler_refdata.py]: it's can be called from diff thread

**Pull request merged**